psiminer-core/src/main/kotlin/labelextractor/MethodCommentLabelExtractor.kt

```diff
@@ -0,0 +1,29 @@
+package labelextractor
+
+import astminer.common.splitToSubtokens
+import com.intellij.psi.PsiComment
+import com.intellij.psi.PsiElement
+import com.intellij.psi.javadoc.PsiDocComment
+import com.intellij.psi.util.PsiTreeUtil
+import psi.language.LanguageHandler
+
+class MethodCommentLabelExtractor : LabelExtractor() {
+    override val granularityLevel = GranularityLevel.Method
+
+    override fun handleTree(root: PsiElement, languageHandler: LanguageHandler): Label? {
+        val methodCommentNodes = PsiTreeUtil.collectElementsOfType(root, PsiComment::class.java)
+
+        val methodDocComments = methodCommentNodes.filterIsInstance<PsiDocComment>()
+            .map { it.descriptionElements }.flatMap { it.map { psiElement -> psiElement.text } }
+        val methodComments = methodCommentNodes.filter { it !is PsiDocComment }.map { it.text }
+        val onlyCommentsText = methodDocComments + methodComments
```

Comment:
There should be an option to use only JavaDoc in this task (and I would expect just-JavaDoc by default).